src/Components/Authentication/Desktop/LoginForm.tsx

```diff
@@ -60,7 +60,7 @@ export class LoginForm extends Component<FormProps, LoginFormState> {
           }
 
           return (
-            <Form onSubmit={handleSubmit}>
+            <Form onSubmit={handleSubmit} data-test-login-form>
```

Comment:
We'll want to follow [this format](https://github.com/damassi/reaction/blob/d43843bf504b980fa8d6862cc3520c39b9c6065e/src/Apps/Order/Components/PaymentPicker.tsx#L270)

Reply:
```suggestion
            <Form onSubmit={handleSubmit} data-test="LoginForm">
```